app/src/main/java/com/arygm/quickfix/ui/authentication/WelcomeScreen.kt

```diff
@@ -0,0 +1,197 @@
+package com.github.se.bootcamp.ui.authentication
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.ui.graphics.graphicsLayer
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+
+
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+@Preview
+fun WelcomeScreen(LoD: Boolean = true) {
+
+    val color1 = if (LoD) Color(0xFFF16138) else Color(0xFF633040)
+    val color2 = if (LoD) Color(0xFF731734) else Color(0xFFB78080)
+    val backgroundColor = if (LoD) Color.White else Color(0xFF282828)
```

Comment:
To improve readability and maintainability , it would be beneficial to create a class or object to centralize the colors as constants. This approach makes it easier to manage colors and avoids repetitive code